tck/src/main/java/com/github/bsideup/liiklus/positions/PositionsStorageTests.java

```diff
@@ -0,0 +1,105 @@
+package com.github.bsideup.liiklus.positions;
+
+import lombok.val;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import reactor.core.publisher.Flux;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public interface PositionsStorageTests extends PositionsStorageTestSupport {
```

Comment:
could you please split this into a few (akin Records Storage TCK)? 
e.g. `GroupsTest`, `PersistenceTest`, etc. 

It will also make it easy to add more tests & categories in future